cli/dcoscli/package/main.py

```diff
@@ -401,7 +401,14 @@ def _list(json_, endpoints, app_id, package_name):
 
             results.append(pkg_info)
 
-    emitting.publish_table(emitter, results, tables.package_table, json_)
+    if results:
+        emitting.publish_table(emitter, results, tables.package_table, json_)
+    else:
+        msg = ("There are currently no installed packages. "
+               "Please use `dcos package install` to install a package.")
+        if json_:
```

Comment:
It would break scripts anyway as soon as they try to parse the contents.

Use a DCOSException so it goes to stderr.